src/ir/comp.rs

```diff
@@ -1694,6 +1718,71 @@ impl CompInfo {
 
         true
     }
+
+    // Check whether a bit field's bit width expression contains any references
+    // to template parameters. If so, we don't evaluate its width.
+    fn is_field_bit_width_template_dependent(cursor: &clang::Cursor) -> bool {
+        use clang_sys::*;
+
+        if !cursor.is_bit_field() {
```

Comment:
Maybe `debug_assert!(cursor.is_bit_field())`? Caller checks, but if we can restore the printing moving this to `clang::Cursor` let's do that I think.